spec/models/item_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+describe Item, type: 'model'  do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.